### DISCUSSION TITLE
docker本地部署没有安装步骤，直接显示登录页面，没有账号密码

### DISCUSSION THREAD

**Discussion by williechan91** [langgenius/dify#3649] [General]
![image](https://github.com/langgenius/dify/assets/161340088/a6387f98-700e-4b8a-9ef3-ed8e6ddf9203)
大家好，我刚刚部署完，直接就是让我登录了，也没有创建账号密码，这是什么情况呢？

**Comment by LSXNN** [comment]
试试打开10.0.0.1:3060/install链接创建账号密码

**Reply by williechan91** [reply]
![image](https://github.com/langgenius/dify/assets/161340088/3cf8da87-265b-417c-bdfc-48ff696820fd)
昨天有想到这个，但是install这个页面一直就这样

**Reply by saminbj** [reply]
+1

**Reply by LSXNN** [reply]
看看db数据库是否已经正常启动？

**Comment by crazywoola** [comment]
看文档，文档上面写过了，转圈是因为环境变量没有配置。

**Reply by elttilz** [reply]
怎么配置，如果我没有域名，只有ip地址能不能配置

**Reply by crazywoola** [reply]
不是的，你这个版本有点老了，后面的版本我特意加上了一个链接去初始化账号，在 privacy policy 下面

**Reply by johnfey** [reply]
> 不是的，你这个版本有点老了，后面的版本我特意加上了一个链接去初始化账号，在隐私政策下面

超级管理员的密码忘了怎么办？给邮箱发的邮件收不到

**Reply by crazywoola** [reply]
> > 不是的，你这个版本有点老了，后面的版本我特意加上了一个链接去初始化账号，在隐私政策下面
> 
> 超级管理员的密码忘了怎么办？给邮箱发的邮件收不到

看看文档如何重置密码
